fastfield_codecs/src/column.rs

```diff
@@ -31,13 +31,21 @@ pub trait Column<T: PartialOrd = u64>: Send + Sync {
         }
     }
 
-    /// Return the positions of values which are in the provided range.
+    /// Get the positions of values which are in the provided value range.
+    ///
+    /// Note that position == docid for single value fast fields
     #[inline]
-    fn get_between_vals(&self, range: RangeInclusive<T>) -> Vec<u64> {
+    fn get_positions_for_value_range(
+        &self,
+        value_range: RangeInclusive<T>,
+        doc_id_range: Range<u64>,
```

Comment:
We should pass a `&mut Vec<u64>` here. (And evenutally a `Vec<u32>`)

Reply:
I agree, I wanted to keep the PR small

Reply:
I don't think it makes the code longer... but ok... Can you keep that in your todo list or open a ticket?